lib/spells/TargetCondition.cpp

```diff
@@ -277,16 +265,11 @@ class ReceptiveFeatureCondition : public TargetConditionItemBase
 
 private:
 	CSelector selector;
-	std::string cachingString;
+	std::string cachingString = "type_RECEPTIVE";
```

Comment:
Please keep initializers in one place. It is weird to have selector initialized in constructor and cachingString in-place

Reply:
all to fields done